aea/aea.py

```diff
@@ -34,12 +35,15 @@
 from aea.protocols.default.message import DefaultMessage
 from aea.registries.filter import Filter
 from aea.registries.resources import Resources
-from aea.skills.error import ERROR_SKILL_ID
 from aea.skills.error.handlers import ErrorHandler
 from aea.skills.tasks import TaskManager
 
 logger = logging.getLogger(__name__)
 
+ERROR_SKILL_ID = PublicId(
```

Comment:
There was a circular import issue with this. Now looking at it more carefully, I think we need to make this configurable anyway. The user should be forced to specify the error handler (by default we use ours).

Reply:
Ok, but maybe not. The error skill should be the same for every agent (although we can't enforce that anyway).